tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -102,6 +102,145 @@ spec:
 			retry.UntilSuccessOrFail(t, func() error {
 				return expectVirtualServiceStatus(t, ns, false)
 			})
+			// Apply bad config (ServiceEntry missing address)
+			t.ConfigIstio().YAML(ns.Name(), `
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: vm-1
+spec:
+  hosts:
+  - dummy.github.com
+  addresses:
+  - 1.1.1.1
+  location: MESH_EXTERNAL
+  ports:
+  - name: tcp
+    number: 22
+	protocol: TCP
+`).ApplyOrFail(t)
```

Comment:
This has an addresses, it looks like there is a yaml indentation problem to my eye. Is comment just old?

In practice I suspect a real kube-api would just flat prevent someone from writing malformed yaml though

Reply:
I have to push some updates I have locally. But also based on our slack conversation... I'm not sure that this is actually flexing the analyzer itself. May end up abandoning this approach in favor of a more direct way of testing what we care about here.